### PR TITLE
UI for Multi images barcode detection

### PR DESCRIPTION
**Pull request by scan-mayank**
- [x] Added UI item for opening the gallery and detecting multiple barcodes.
- [x] Added listing page for showing the images and barcode text

### CODE REVIEW

**Comment by scan-mayank** [comment]
Please don't merge this PR.
This PR has a dependency on this SDK PR: https://github.com/doo/scanbot-sdk-flutter/pull/106
So first the SDK will merge and then this current PR can proceed ahead

File: lib/ui/barcode_preview_multi_image.dart [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+import 'package:flutter/material.dart';
+import 'package:scanbot_sdk/barcode_scanning_data.dart';
+import 'package:scanbot_sdk_example_flutter/ui/pages_widget.dart';
+
+import '../main.dart';
+import 'barcode_preview.dart';
+
+class MultiImageBarcodesResultPreviewWidget extends StatelessWidget {
+  final List<BarcodeScanningResult> lstPreviewItems;
```
Comment (rutkayb) [inline lib/ui/barcode_preview_multi_image.dart:11]: Consider using full word instead of an abbreviation

**Review by Sarbjyotsingh — CHANGES_REQUESTED** [review]
Use` flutter format .`

File: lib/main.dart [hunk 2]
```diff
@@ -444,6 +451,49 @@ class _MainPageWidgetState extends State<MainPageWidget> {
     }
   }
 
+  /// Detect barcodes from multiple still images
+  Future<void> _detectBarcodesOnImages() async {
+    if (!await checkLicenseStatus(context)) {
+      return;
+    }
+    try {
+      List<Uri> lstUris = List.empty(growable: true);
```
Comment (Sarbjyotsingh) [inline lib/main.dart]: Refactor `lstUris` -> `listUris`

Comment (Sarbjyotsingh) [inline lib/main.dart]: `List.empty(growable: true)` -> `<Uri>[] `

This syntax is More familiar for flutter developer

File: lib/ui/barcode_preview_multi_image.dart [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        body: ConstrainedBox(
+          constraints: BoxConstraints(maxWidth: MediaQuery
+              .of(context)
+              .size
+              .width, maxHeight: MediaQuery
+              .of(context)
+              .size
+              .height),
+          child: ListView.builder(
+              // shrinkWrap: true,
```
Comment (Sarbjyotsingh) [inline lib/ui/barcode_preview_multi_image.dart:43]: Remove Commented Code

File: lib/ui/barcode_preview_multi_image.dart (hunk 1 above)
Comment (scan-mayank) [reply lib/ui/barcode_preview_multi_image.dart]: Fixed 👍

**Review by Sarbjyotsingh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**